src/sidebar/components/menu-item.js

```diff
@@ -60,18 +64,25 @@ export default function MenuItem({
   const hasSubmenuVisible = typeof isSubmenuVisible === 'boolean';
   const isRadioButtonType = typeof isSelected === 'boolean';
 
+  useEffect(() => {
+    return () => {
+      // unmount
+      clearTimeout(focusTimer.current);
```

Comment:
I assume this does not throw an exception or display a warning if `focusTimer.current` is `null`?

Reply:
correct